src/main/java/com/example/yiseul/converter/MemberConverter.java

```diff
@@ -0,0 +1,54 @@
+package com.example.yiseul.converter;
+
+import com.example.yiseul.domain.Member;
+import com.example.yiseul.dto.member.MemberCreateRequestDto;
+import com.example.yiseul.dto.member.MemberCursorResponseDto;
+import com.example.yiseul.dto.member.MemberPageResponseDto;
+import com.example.yiseul.dto.member.MemberResponseDto;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class MemberConverter {
+
+    public static Member convertMember(MemberCreateRequestDto createRequestDto) {
+
+        return new Member(createRequestDto.name(), createRequestDto.age(), createRequestDto.hobby());
+    }
+
+    public static MemberResponseDto convertMemberResponseDto(Member member) {
+
+        return new MemberResponseDto(member.getId(),
+                member.getName(),
+                member.getAge(),
+                member.getHobby(),
+                member.getCreatedAt(),
+                member.getCreatedBy());
+    }
```

Comment:
p5; 

개인 취향이지만 저는 괄호를 아래로 빼는 것이 좀 더 명확해보이더라구요 ㅎ 
```suggestion
        return new MemberResponseDto(
                member.getId(),
                member.getName(),
                member.getAge(),
                member.getHobby(),
                member.getCreatedAt(),
                member.getCreatedBy()
         );
    }
```

Reply:
확실히 괄호를 아래로 빼서 하는 것이 가독성이 좋네요👍 감사합니다!